etna/records/tests/test_views.py

```diff
@@ -722,3 +725,100 @@ def test_invalid_response_from_kong_raises_404(self):
         response = self.client.get("/records/images/C123456/11000000000000000000/")
 
         self.assertEquals(response.status_code, 404)
+
+
+class RecordDetailBackToSearchTest(TestCase):
+    def setUp(self):
+        responses.add(
+            responses.GET,
+            "https://kong.test/data/fetch",
+            json=create_response(
+                records=[
+                    create_record(iaid="C13359805"),
+                ]
+            ),
+        )
+
+        self.record_detail_url = reverse(
+            "details-page-machine-readable", kwargs={"iaid": "C13359805"}
+        )
+
+        self.expected_button_link_gen_value_fmt = '<a class="cta-primary-panel__link" href="{back_to_search_url}" data-link-type="Link" data-link="Back to search results" data-component-name="Navigation">\n\n\n\n<svg\n    class="icon icon--chevron-left cta-primary-panel__link-icon"\n    aria-hidden="true"\n    focusable="false"\n>\n    <use xlink:href="#chevron-left" href="#chevron-left"></use>\n</svg>\nBack to search results</a>'
```

Comment:
What happens if we just add the opening tag here? 
```suggestion
        self.expected_button_link_gen_value_fmt = '<a class="cta-primary-panel__link" href="{back_to_search_url}" data-link-type="Link" data-link="Back to search results" data-component-name="Navigation">'
```

Reply:
Missed this bit as the earlier implementation had a label in the button which was changing. Fixed.